hack/test-integration.sh

```diff
@@ -96,16 +101,39 @@ loop="${TIMES:-1}"
 # hack/test-integration.sh Template*
 # hack/test-integration.sh "(WatchBuilds|Template)"
 tests=( $(go run "${OS_ROOT}/hack/listtests.go" -prefix="${OS_GO_PACKAGE}/${package}.Test" "${testexec}" | grep -E "${1-Test}") )
+
+if [[ -n "${junit_report}" ]]; then
```

Comment:
The convention is to use `$JUNIT_REPORT` to set these like: https://github.com/openshift/origin/blob/master/hack/test-cmd.sh#L79-L82